ClassicLootManager/Modules/Rosters/RosterManager/LedgerEntries.lua

```diff
@@ -511,6 +512,29 @@ function RosterDynamicItemValueExpvar:fields()
     return RosterDynamicItemValueExpvarFields
 end
 
+-- ---------------------------- --
+-- RosterDynamicItemValueCustomExpression --
+-- ---------------------------- --
+function RosterDynamicItemValueCustomExpression:new(rosterUid, custom_expression)
+    local o = LogEntry.new(self);
+    o.r = tonumber(rosterUid) or 0
+    o.m = custom_expression or "0"
```

Comment:
- Hard limit custom expression length to `100` characters (substr) in here.
- Hard cast custom_expression to string (needed for the substr)
- Be sure to error out if limit is exceeded in validation function